src/components/ColorPicker/ColorPicker.visual.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import { storiesOf } from '@storybook/react';
+import { VisualContainerElement } from '../../../test/visual/VisualContainerElement';
+import { ColorPicker } from './';
+
+class ColorPickerVisual extends React.Component<any> {
+  render() {
+    return (
+      <VisualContainerElement>
+        <ColorPicker {...this.props} onChange={color => console.log(color)} />
```

Comment:
We should have a visual test with at least one color

Reply:
done